examples/service-catalog/service-catalog.yaml

```diff
@@ -0,0 +1,370 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: service-catalog
+objects:
+
+- kind: ClusterRoleBinding
```

Comment:
> Doesn't the TSB trust callers to provide information about who is requesting a particular resource? I remember having a special permission to allow psuedo-impersonation for that purpose, but if the SC lacks the ability to authenticate, how are you going to restrict it?

we aren't/won't/can't for now.  The TSBs ability to impersonate is intended to be shielded by limiting who can call the TSB, but until the SC can provide its auth, we can't actually limit who can call the TSB.

> Why is necessary? The SC should have a direct (never touching aggregator) loopback connection to itself which has full permissions.

good question.  @pmorie i think we can dump this one?

> Are you exposing etcd as as service in your namespace? I did not expect that. Doesn't that open etcd to attack from anyone on the cluster?

no, there's no service for it, it's just running in a sidecar container.  you'd have to ask @pmorie why they need this, but it's somehow tied to them doing leader election.